cli/command/image/build_session.go

```diff
@@ -28,6 +28,9 @@ import (
 const clientSessionRemote = "client-session"
 
 func isSessionSupported(dockerCli command.Cli) bool {
+	if versions.GreaterThanOrEqualTo(dockerCli.Client().ClientVersion(), "1.39") {
```

Comment:
This should check daemon version, not CLI.

Reply:
Daemon could be version 1.40, but the client downgraded to 1.10; in that case the API version that's used is 1.10, and it's not supported